src/Core/Checkout/Order/SalesChannel/OrderRoute.php

```diff
@@ -71,7 +71,11 @@ public function load(Request $request, SalesChannelContext $context, Criteria $c
         if ($context->getCustomer()) {
             $criteria->addFilter(new EqualsFilter('order.orderCustomer.customerId', $context->getCustomerId()));
         } elseif ($deepLinkFilter === null) {
-            throw CartException::customerNotLoggedIn();
+            // @deprecated tag:v6.8.0 - remove this if block
+            if (!Feature::isActive('v6.8.0.0')) {
+                throw CartException::customerNotLoggedIn(); // @phpstan-ignore-line shopware.domainException
```

Comment:
```suggestion
                throw CartException::customerNotLoggedIn(); // @phpstan-ignore shopware.domainException
```

if you want to ignore specific identifier, that is the correct syntax :wink: